src/templates/index.html

```diff
@@ -4,46 +4,77 @@
-    <form action="/change_username" method="post">
-        <label for="username">Enter New Username:</label>
-        <input type="text" id="newusername" name="newusername" required>
-        <input type="submit" value="Submit">
-    </form>
-
-    <h1>Here to change your courses: </h1>
-    <p>You have {{ courses|length }} course(s)</p>
-    <p>* Press the X on the right of a course to remove it</p>
-
-    <ul>
-        {% for course in courses %}
-        <li style="float: left">{{ loop.index }}. {{ course }}</li>
-        <form action="/remove_course" method="post">
-            <input type="hidden" name="index" value="{{ loop.index - 1 }}">
-            <input type="hidden" name="course" value="{{ course }}">
-            <input type="submit" value="X">
-        </form>
-        {% endfor %}
-    </ul>
-
-    <form action="/add_course" method="post">
-        <label for="newcourse">Add a New Course:</label>
-        <input type="text" id="newcourse" name="newcourse" required>
-        <input type="submit" value="Add">
-    </form>
+    {% include 'header.html' %}
+    <div class="banner-container" style="background-color: antiquewhite;">
+        <div id="banner" class="banner-bar">
+            <img style="border-radius: 0; display: inline-block;" class="banner-image"
+                src="{{url_for('static', filename='fight.png')}}" />
+            <div>Lets start today's work with your course buddy!</div>
+        </div>
+    </div>
+    <div class="course-panel-container">
+        <div class="course-list">
+            <div class="panel-course" style="margin: 5px;">
+                <div style="display: flex; margin: 5px;">
+                    <h2>My Courses </h2>
+                    <h3>({{courses|length}})</h3>
+                </div>
+
+                <div class="courses" style="display: table; width: 256px;  border-spacing: 10px;">
+                    {% for course in courses %}
+                    <div id="{{ course.0.1 }}"" class=" course-detail">
+                        <div class="course-image">
+                            <img style="vertical-align:middle " src="{{url_for('static', filename='books.png')}}" />
+                        </div>
+                        <div class="course-progress">
+                            <div class="progress-bar"></div>
+                            <!-- dynamicly change progress bar -->
+                            <!-- <div class="progress-bar" style="width: {{ course_completion_percentage }}%;"></div> -->
+                        </div>
+                        <div class="course-details">
+                            <div class="course-title">{{course}}</div>
+                            <div class="course-desc">This a course in level 4, aims to learn the sicentific computation
+                            </div>
+                            <div class="instructor">By Prof Dr NED NEDIALKOV</div>
+                        </div>
```

Comment:
I think these course details should be extracted from the course outline right? (Maybe create an issue to change this part after the uploading pdf feature is added but for now could use this as an example)